### PR TITLE
EPMRPP-37459 Dashboard. Widgets

### PR DESCRIPTION
**Pull request by AmsterGet**
- Request for add widget to dashboard
- Fix request for delete widget
- Change widgets parameters according to new API
- Fix close widget wizard modal
- Small UI fixes

**Need more fixes for some created widgets**

### CODE REVIEW

File: app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { redirect } from 'redux-first-router';
 import { defectLinkSelector, statisticsLinkSelector } from 'controllers/testItem';
 import * as COLORS from 'common/constants/colors';
 import { STATS_PASSED } from 'common/constants/statistics';
+import { CHART_MODES } from 'pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants';
```
Comment (DiscoElevator) [inline app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx]: Looks strange that widget depends on something from creation modal. I think it's better to move this constant to an other place.

File: app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx [hunk 2]
```diff
@@ -170,15 +155,19 @@ export class Widget extends Component {
...
-            {this.state.loading && <SpinningPreloader />}
-            {Chart && (
-              <Chart
-                widget={widget}
-                isFullscreen={this.props.isFullscreen}
-                container={this.node}
-                observer={this.props.observer}
-              />
-            )}
+            {(this.state.loading && <SpinningPreloader />) ||
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx]: It's better to split these conditions. Maybe create some class methods or just variables.
It's really difficult to read such nested conditions.

File: app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx]: Done

File: app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx]: Done. Сlass method `getWidgetContent` has been added.

**Pull request merged**